desktop/app/menu-helper.js

```diff
@@ -35,7 +35,15 @@ export default function makeMenu (window) {
         {label: 'Minimize', accelerator: 'CmdOrCtrl+M', role: 'minimize'},
         {label: 'Close', accelerator: 'CmdOrCtrl+W', role: 'close'},
         {type: 'separator'},
-        {label: 'Bring All to Front', role: 'front'}
+        {label: 'Bring All to Front', role: 'front'},
```

Comment:
Reload is probably useful on prod too. Example: Slack uses it on prod, and it could be a way to know if reloading something fixes it.